src/components/ComputeModal.js

```diff
@@ -298,12 +299,14 @@ export const ComputeModalBase = ({ onDismiss, onSuccess, runtimes, persistentDis
     const cloudService = runtimeConfig?.cloudService
     const numberOfWorkers = runtimeConfig?.numberOfWorkers || 0
     const gpuConfig = runtimeConfig?.gpuConfig
+    const tool = currentRuntimeDetails?.labels.tool
```

Comment:
```suggestion
    const tool = currentRuntimeDetails?.labels?.tool
```
so it doesn't explode in case `currentRuntimeDetails` doesn't have `labels`.